swap/src/protocol/alice/swap.rs

```diff
@@ -1,19 +1,19 @@
 //! Run an XMR/BTC swap in the role of Alice.
 //! Alice holds XMR and wishes receive BTC.
-use anyhow::Result;
+use anyhow::{bail, Result};
 use async_recursion::async_recursion;
 use futures::{
     future::{select, Either},
     pin_mut,
 };
 use rand::{CryptoRng, RngCore};
 use std::sync::Arc;
-use tracing::info;
+use tracing::{error, info};
 use uuid::Uuid;
 
 use crate::{
```

Comment:
```suggestion
use uuid::Uuid;
use crate::{
```

This newline is problematic for IDE. Let's follow `rustfmt`'s output (which does **not** separate the imports).

Reply:
https://github.com/comit-network/xmr-btc-swap/pull/153